.pre-commit-config.yaml

```diff
@@ -8,25 +6,16 @@ repos:
     -   id: end-of-file-fixer
     -   id: check-yaml
     -   id: check-added-large-files
-
-    # Clang-format for C++
-    # This brings in a portable version of clang-format.
-    # See also: https://github.com/ssciwr/clang-format-wheel
   - repo: https://github.com/pre-commit/mirrors-clang-format
     rev: v18.1.8
     hooks:
     - id: clang-format
       types_or: [c++, c]
-
-    # CMake linting and formatting
   - repo: https://github.com/BlankSpruce/gersemi
     rev: 0.15.1
     hooks:
     - id: gersemi
       name: CMake linting
-
-    # Markdown linting
-    # Config file: .markdownlint.yaml
```

Comment:
There are multiple CMake linters using different file names for their linting file. This is useful for alleviate confusion when someone brings in another CMake lint config.

Reply:
You can see an instance of this happening here:
https://github.com/bemanproject/inplace_vector/pull/14/files#r1842893309